atlassian/rest_client.py

```diff
@@ -201,15 +201,15 @@ def get(self, path, data=None, flags=None, params=None, headers=None, not_json_r
         """
         response = self.request('GET', path=path, flags=flags, params=params, data=data, headers=headers,
                                 trailing=trailing)
-        if self.advanced_mode:
-            return response
         if not_json_response:
             return response.content
+        if self.advanced_mode:
+            return response
```

Comment:
Please pay attention!
In this place, the order of lines was messed and the 'else' block below ran in wrong cases

Reply:
Could you reverse that change, please?